docs/mesh_results.md

```diff
@@ -10,6 +10,13 @@ MTI      | 0.49
 Version   | Approach    | Micro f1 in Pubs | Micro f1 in Grants | Description
 --------- | ----------- | ---------------- | ------------------ | -------
 2020.07.0 | tfidf-sgd   | 0.56             | 0.69               | tfidf-svm, bigrams, regularisation of 1e-9, removal of words appearing less than 5 times
+2020.9.0  | cnn         | 0.64             |                    | 1e-3 lr, 0.1 dr, 1e-7 l2, 4 layers, attention
+2020.9.1  | cnn         | 0.63             |                    | 1e-3 lr, 0.1 dr, **1e-8 l2**, 4 layers, attention
+2020.9.2  | cnn         | 0.64             |                    | 1e-3 lr, 0.1 dr, 1e-7 l2, 4 layers, ~~attention~~
+2020.9.3  | cnn         | 0.57             |                    | 1e-3 lr, 0.1 dr, 1e-7 l2, **0 layers**, attention
+2020.9.4  | cnn         | 0.62             |                    | 1e-3 lr, **0 dr**, 1e-7 l2, 4 layers, attention
```

Comment:
We don't have any result on grants?

Reply:
I will add separately. The problem is the format of the data that needs a bit of preprocessing to run the results. Will do very soon